File: server/my-saloon/my-saloon/src/main/java/com/rmaciel/mysaloon/controllers/dtos/CustomerDTO.java
package com.rmaciel.mysaloon.controllers.dtos;

import com.rmaciel.mysaloon.models.Customer;

public class CustomerDTO {
    private Long id;
    private String fullname;
    private String residencialPhone;
    private String cellphone;
    private String indicatedBy;
    private String professionalEngagedName;


    public CustomerDTO(Customer customer) {
        this.id = customer.getId();
        this.fullname = customer.getFullname();
        this.residencialPhone = customer.getResidencialPhone();
        this.cellphone = customer.getCellphone();
        this.indicatedBy = customer.getIndicatedBy();
        this.professionalEngagedName = customer.getProfessionalEngaged().getName();
    }


    public Long getId() {
        return this.id;
    }

    public String getFullname() {
        return this.fullname;
    }

    public String getResidencialPhone() {
        return this.residencialPhone;
    }

    public String getCellphone() {
        return this.cellphone;
    }

    public String getIndicatedBy() {
        return this.indicatedBy;
    }

    public String getProfessionalEngagedName() {
        return this.professionalEngagedName;
    }

}   

File: client/src/dtos/CustomerDTO.js
export default class CustomerDTO {
    constructor(fullname, residencialPhone, cellphone, indicatedBy, professionalEngaged, id='') {
        this.fullname = fullname;
        this.residencialPhone = residencialPhone;
        this.cellphone = cellphone;
        this.indicatedBy = indicatedBy;
        this.professionalEngagedID = professionalEngaged;
        this.id = id;
    }

}